src/signing.js

```diff
@@ -7,6 +7,8 @@
 
 var actualMethods = {};
 
+export var fastSigning;
```

Comment:
I think there is no way to achieve this because:
* `export` statments can be only occur at the top of the file. 
* Setting `fastSigning ` value depends on the flow of the program.

So after thinking about it, there are two ways to do it:
* `Object.defineProperty` (ugly).
* Create a method (`isFastSigningAvailable()`?) that will return a copy of `fastSigning` value.